xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/main/java/org/xwiki/rendering/macro/AbstractMacro.java

```diff
@@ -257,15 +257,15 @@ protected void setDefaultCategory(String defaultCategory)
     }
 
     /**
-     * Helper to get the proper metadata for unchanged content (i.e. content that has not gone through a
-     * Transformation).
+     * Helper to get the proper metadata for non generated content (i.e. content that has not gone through a
+     * Transformation). This content can be used for inline editing.
      *
      * @return the new metadata with the content type for the content represented as a string (e.g.
-     *         {@code java.util.List< org.xwiki.rendering.block.Block >} for content of type {@code Listw<Block>}
+     *         {@code java.util.List< org.xwiki.rendering.block.Block >} for content of type {@code List<Block>}
      * @since 10.10RC1
```

Comment:
Wrong since.

Reply:
why? the method signature has no changed AFAICS.